modules/ROOT/pages/provisioning-exoscale.adoc

```diff
@@ -0,0 +1,64 @@
+= Provisioning Fedora CoreOS on Exoscale
+
+This guide shows how to provision new Fedora CoreOS (FCOS) instances on https://exoscale.com[Exoscale] Cloud Hosting.
+
+== Prerequisites
+
+Before provisioning a FCOS instance, you can use or not an Ignition configuration file containing your customizations. If you need one, see xref:producing-ign.adoc[Producing an Ignition File].
```

Comment:
This is a funny sentence :)

I see that that in other similar pages we have a "must" which is probably too strong. What about "you may want to have at hand an Ignition configuration ..."?

Reply:
5b181e5